sdk/deviceupdate/iot-device-update-rest/samples-dev/getDevice.ts

```diff
@@ -19,17 +19,60 @@ const instanceId = process.env["INSTANCE_ID"] || "";
 
 async function main() {
   console.log("== Get device ==");
-  const device = process.env["DEVICEUPDATE_DEVICE"] || "";
+  const groupId = process.env["DEVICEUPDATE_DEVICE_GROUP"] || "";
 
   const credentials = new DefaultAzureCredential();
 
   const client = DeviceUpdate(endpoint, credentials);
 
-  const result = await client
-    .path("/deviceUpdate/{instanceId}/management/devices/{deviceId}", instanceId, device)
+  console.log("Get various device management information from Device Update for IoT Hub...");
+
+  console.log("\nDevices:");
+  const devicesResult = await client
+    .path("/deviceUpdate/{instanceId}/management/devices", instanceId)
+    .get();
+  if (isUnexpected(devicesResult)) {
```

Comment:
nit: it would improve readability if you add an empty like before and after blocks